Documentation/using-rkt-with-systemd.md

```diff
@@ -202,6 +202,86 @@ Jul 30 12:24:50 locke-work systemd[1]: Listening on My socket-activated app's so
 
 Now, a new connection to port 8080 will start your container to handle the request.
 
+### Bidirectionally proxy local sockets to another (possibly remote) socket.
+
+`rkt` also supports [socket-proxyd service][systemd-socket-proxyd]. So, like in
```

Comment:
`rkt` also supports the [socket-proxyd service][systemd-socket-proxyd]. Much like socker activation, with socket-proxyd systemd provides a listener on a given port on behalf of a container, and starts the container when a connection is received. Socket-proxy listening can be useful in environments that lack native support for socket activation. The LKVM stage1 flavor is an example of such an environment.